silx/io/fabioh5.py

```diff
@@ -53,6 +53,36 @@
 _logger = logging.getLogger(__name__)
 
 
+_fabio_extensions = set([])
+
+
+def available_extensions():
```

Comment:
Why not calling it `supported_extensions` as the `silx.io` one?

Reply:
It is not the same output, but yes, it can be called the same